packages/e2e-tests/package.json

```diff
@@ -38,6 +38,6 @@
     "jest-puppeteer": "^9.0.2"
```

Comment:
Let's update this too.

Ditto for `jest-environment-puppeteer` and `expect-puppeteer

Reply:
Running `npm update jest-puppeteer --save` and `npm update jest-environment-puppeteer --save ` did not make any changes